### ISSUE TITLE
Fatal error from /launch route on Heroku

### ISSUE THREAD

**Issue by jessesnyder** [Dallinger/Dallinger#2130]
## Current Behavior
Call  to /launch route fails on Heroku:

```
2020-04-27T16:26:47.796659+00:00 app[api]: Scaled to console@0:Hobby web@1:Hobby worker@0:Hobby by user admin@example.com
2020-04-27T16:26:47.779216+00:00 app[api]: Deploy 535b8879 by user admin@example.com
2020-04-27T16:26:47.779216+00:00 app[api]: Release v11 created by user admin@example.com
2020-04-27T16:26:55.889380+00:00 heroku[web.1]: State changed from starting to crashed
2020-04-27T16:26:55.893186+00:00 heroku[web.1]: State changed from crashed to starting
2020-04-27T16:26:55.743406+00:00 app[web.1]:
2020-04-27T16:26:55.743421+00:00 app[web.1]: Error: '//dlgr-e043c8aa.herokuapp.com:56029' is not a valid port number.
2020-04-27T16:26:55.743426+00:00 app[web.1]:
2020-04-27T16:27:00.000000+00:00 app[api]: Build succeeded
2020-04-27T16:27:05.834200+00:00 heroku[web.1]: State changed from starting to crashed
2020-04-27T16:27:05.574234+00:00 app[web.1]:
2020-04-27T16:27:05.574254+00:00 app[web.1]: Error: '//dlgr-e043c8aa.herokuapp.com:41770' is not a valid port number.
2020-04-27T16:27:05.574261+00:00 app[web.1]:
2020-04-27T16:27:05.673011+00:00 app[api]: Scaled to console@0:Hobby web@1:Hobby worker@1:Hobby by user admin@example.com
...
2020-04-27T16:27:19.951923+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=POST path="/launch" host=dlgr-e043c8aa.herokuapp.com request_id=028c0190-0938-426a-8bab-bde9ae071bba fwd="24.143.98.11" dyno= connect= service= status=503 bytes= protocol=https
```

## Steps to Reproduce (for bugs)
`dallinger sandbox` using the Batlett demo.

**Comment by jessesnyder** [comment]
I think this problem is similar to one I encountered with `PORT` a few years ago:
https://github.com/Dallinger/Dallinger/commit/d3fd5078796b4be85b0603f8dd9eb6bc44e90fd7

**Comment by jessesnyder** [comment]
@MatthewWilkes The changes to allow acquiring `host` configuration from the `HOST` environment variable broke Heroku deployments. I suspect the problem might be a sibling of one I encountered with `port`/`PORT` (see https://github.com/Dallinger/Dallinger/commit/d3fd5078796b4be85b0603f8dd9eb6bc44e90fd7 as I mentioned above).

I've made a hasty PR which basically reverts that change, since things seem to work without it, but I'm sure you had reasons for adding the `HOST`/`host` aliasing, so wanted to loop you in.

The PR: https://github.com/Dallinger/Dallinger/pull/2131
